### PR TITLE
feature: load plugins relative to the current working directory

### LINKED ISSUE #42: Load plugins in cwd
When a CLI tool which uses this module is installed globally, it's supposed to find plugins in current working directory instead of the directory where the CLI is installed.

If I'm not mistaken, change `require(plugin)` to `requireCwd(plugin)` (https://github.com/sindresorhus/req-cwd) could solve this.

### PR DESCRIPTION
**Pull request by sinisterstumble**
## Proposed changes

In the spirit zero configuration (#0CJS) trend postcss-load-plugins should be able to require plugins relative to the `ctx.cwd`.

Closes #42 

* There are few lint issues unrelated to the code I submitted, those I have not touched.

## Types of changes

- ~~Bug (non-breaking change which fixes an issue)~~
- [x] Feature (non-breaking change which adds functionality)
- [~~Breaking Change (fix or feature which changes existing functionality)~~

## Checklist

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- ~~I have added necessary documentation (if appropriate)~~
- ~~Any dependent changes are merged and published in downstream modules~~

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -4,15 +4,39 @@
...
+  try {
+    return require(plugin)
+  } catch (_err) {
+    err = _err
+  }
+
+  try {
+    return require(resolveFrom(cwd, plugin))
+  } catch (_err) {}
+
```
Comment (Scrum) [inline lib/plugins.js:29]: it seems to me [flatry](https://github.com/ymatuhin/flatry) will be appropriate here

File: package.json [hunk 2]
```diff
@@ -2,19 +2,22 @@
   "name": "postcss-load-plugins",
   "version": "2.3.0",
   "description": "Autoload Plugins for PostCSS",
-  "engines": { "node": ">=0.12" },
+  "engines": {
+    "node": ">=0.12"
```
Comment (Scrum) [inline package.json:6]: can be raised to 4+

**Review by Scrum — APPROVED** [review]

File: lib/plugins.js (hunk 1 above)
Comment (sinisterstumble) [reply lib/plugins.js:29]: Didn't want to add an extra dependency ;-)

**Comment by sinisterstumble** [comment]
@GitScrum @michael-ciniawsky let's get this merged?

**Comment by Scrum** [comment]
@michael-ciniawsky I need access from npm to publish this

**Comment by Scrum** [comment]
@michael-ciniawsky ping

**Comment by Scrum** [comment]
@michael-ciniawsky ping

**Comment by sinisterstumble** [comment]
@Scrum @michael-ciniawsky ping

**Comment by Scrum** [comment]
@l5x  sorry, I still do not have access.

**Comment by michael-ciniawsky** [comment]
@l5x This is supported  in [`postcss-load-config >= v2.0.0`](https://github.com/michael-ciniawsky/postcss-load-config/releases/tag/v2.0.0) via [`9745bf0`](https://github.com/michael-ciniawsky/postcss-load-config/commit/9745bf0) now. Since I'm going to archive this repo here soon I'm closing this PR. Thx

**Pull request closed without merging**